File: CWEB 2116 - Homework 2/OperationsApplication/TemperatureApp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OperationsApp
{
	class TemperatureApp
	{
		static void Main(string[] args)
		{
			Console.WriteLine("Temperature Conversion");
			Console.WriteLine("----------------------");
			PerformTemperatureConversion conversion1 = new PerformTemperatureConversion();
			conversion1.performTemperatureConversion();
		}
	}
}

File: CWEB2116RD2/src/operationsApplication/TemperatureApp.java
package operationsApplication;

public class TemperatureApp {

	public static void main (String[] args) {
		
		System.out.println("Temperature Conversion");
		System.out.println("----------------------");
		PerformTemperatureConversion conversion = new PerformTemperatureConversion();
		conversion.performConversion();
	}
}
